content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -1,41 +1,51 @@
-==============================
-Testing and running a campaign
-==============================
+=========================
+Testing/running campaigns
+=========================
 
 Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
 them.
 
-First, open the :guilabel:`Marketing Automation` application and click on a campaign. Make sure
-the campaign already has activities configured on it (or build a campaign by following the
-directions here on :doc:`this documentation <workflow_activities>`). To start a test, click the
-:guilabel:`Launch a Test` button at the top of the template form.
+First, open the :menuselection:`Marketing Automation` application, and select the desired campaign,
+which reveals that campaign's detail form.
+
+On the campaign detail form, make sure the campaign already has activities configured (or build a
+campaign by following the directions here on :doc:`this documentation
+</applications/marketing/marketing_automation/getting_started/workflow_activities>`).
+
+To start a test, click the :guilabel:`Launch a Test` button at the top of the campaign form.
 
 .. image:: testing_running/launch-test.png
    :align: center
    :alt: Launch a test button in Odoo Marketing Automation.
 
-When clicked, a pop-up window appears. In the dropdown field choose a specific record to run the
-test on, or create a brand new record by clicking the :guilabel:`Search More...` link at the bottom
-of the dropdown menu, and then click the :guilabel:`Create` button.
+When clicked, a :guilabel:`Launch a test` pop-up window appears. In the drop-down field, choose an
+existing record (or create a new one) to run the test on. To create a brand new record, click the
+:guilabel:`Search More...` link at the bottom of the drop-down menu, and then click the
+:guilabel:`Create` button.
+
+Doing so reveals a blank contact form, in which the necessary contact information should be entered.
```

Comment:
It's probably best to describe here exactly what the "necessary contact information" is